### PR TITLE
Bogo Sort in Python

### LINKED ISSUE #2207: Bogo Sorting in Python
## 🚀 Feature

Bogo Sort or Moneky Sort a.k.a. Stupid  Sort in Python.

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yup

## Pitch

BogoSort, Monkey Sort, or inefficient sort is notoriously famous for being one of the worst sortings algorithms with an infinite upper bound.

And for this very reason, is quite popular. Though completely useless as an algorithm Is a good learning tool at teaching how inefficient an algo could be.

### PR DESCRIPTION
**Pull request by vybhav72954**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
Yes 

### Description

Added Bogo Sorting in Python along with test case to illustrate the concept.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [ ] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

Fixes #2207

#### Additional

I can implement this in other languages as well if needed

### CODE REVIEW

**Comment by vybhav72954** [comment]
Hey @HarshCasper @atarax665 @anushkrishnav Any changes I need to make?

**Comment by anushkrishnav** [comment]
@vybhav72954 there are some formatting issues pointed out by DeepSource fix that .

**Comment by vybhav72954** [comment]
@anushkrishnav  The error is of Anti Patterns.

I have already enforced PEP-8 guidelines aggressively on my code.

The thing is
```py
"""
"""
```
arent meant for providing test cases of all things. We don't have any specific README for that purpose either. So the only solution is i combine both the test cases in one.I can do that if you want

**Comment by vybhav72954** [comment]
@HarshCasper @anushkrishnav Waiting for updates

File: Python/sort/Bogo_sort.py [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import time
```
Comment (atarax665) [inline Python/sort/Bogo_sort.py:9]: Add a beginner pitch of the algorithm.

**Review by atarax665 — CHANGES_REQUESTED** [review]
Please update the corresponding README too.

**Comment by atarax665** [comment]
@vybhav72954 Any updates?

**Comment by vybhav72954** [comment]
> @vybhav72954 Any updates?

I will do this within an hour or so

**Review by atarax665 — APPROVED** [review]
Just update README.md so that I can merge this PR.

**Comment by atarax665** [comment]
??

**Comment by satyampandeygit** [comment]
@atarax665 I can update readme.md so that it can be merged

**Comment by vybhav72954** [comment]
@atarax665 I am sorry! I was traveling and was unable to work. My bad and thank you for helping here.

**Pull request merged**